### DISCUSSION TITLE
Zotero 7 New Update and Better Bibtex shows error

### DISCUSSION THREAD

**Discussion by zararash** [retorquere/zotero-better-bibtex#2828] [Q&A] [answered]
I am facing with below error immediately after updating Zotero 7. I reinstalled Better Bibtex, but the error still remains.

@jar:file:///C:/Users/zarar/AppData/Roaming/Zotero/Zotero/Profiles/l7mv6619.default/extensions/better-bibtex@iris-advies.com.xpi!/bootstrap.js:40:12
@jar:file:///C:/Users/zarar/AppData/Roaming/Zotero/Zotero/Profiles/l7mv6619.default/extensions/better-bibtex@iris-advies.com.xpi!/bootstrap.js:157:3
_loadScope@chrome://zotero/content/xpcom/plugins.js:161:27
Zotero.Plugins</this.init@chrome://zotero/content/xpcom/plugins.js:56:17

![image](https://github.com/retorquere/zotero-better-bibtex/assets/95956329/0e5707cb-09d8-4d39-85bd-7af53a4b523b)

Any solution?

**Comment by marjomol** [comment]
I'm not getting the error window, but the plug-in is not working for me either as it doesn't appear in the settings of Zotero 7 and it cannot be recognised by the obsidian plug-in where I use it. It also happens since the last update of the beta.

**Reply by retorquere** [reply, marked as answer]
bugs are tracked in issues, not discussions, see #2829
